template/nestJs/.env.example

```
@@ -14,3 +14,5 @@ PAGINATION_PAGE = 1
 PAGINATION_LIMIT = 10
 GLOBAL_PREFIX = '/'
 MOCK_REGEX = '/mock'
+# 7天
+REFRESH_TOKEN_TTL = 604800000
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)